components/PluginCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button, Card, CardActions, CardContent, Link, Typography } from "@mui/material"
+import React from 'react';
+
+interface PluginCardProps {
+    title: string
+    description: string
+    githubLink: string
+}
+
+const PluginCard: React.FC<PluginCardProps> = ({ title, description, githubLink }) => (
+  <Card
+    sx={{
+      height: '15rem',
```

Comment:
Plugin cards now have a height of 15rem so they are all the same.
Can also use 100% but this means rows have different heights. minHeight 15rem with height 100% would accomodate for extra text but would cause different height rows, so reasoning was that this would look wrong and cause a manual update if we add a card with more text.

Reply:
This is great, the different heights were bugging me.